src/FunctionBased/components/TodoContainer.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- Kindly, remove this line since it is not allowed to disable eslint 👍

Reply:
Thank you, I do not seem to find a solution to this